server/views/pages/create/bespokeConditions.njk

```diff
@@ -40,7 +40,7 @@
                     <div class="moj-button-action">
                         {{ govukButton({
                             text: 'Add another bespoke condition',
-                            classes: 'govuk-button--secondary moj-add-another__add-button govuk-!-margin-bottom-4',
+                            classes: 'govuk-button--secondary moj-add-another__add-button custom-add-button govuk-!-margin-bottom-7',
```

Comment:
don't think this custom-add-button class is used?

Reply:
removed unused class name.